### PR TITLE
docker image might already exists (created by tempest barclamp)

### PR DESCRIPTION
**Pull request by jsuchome**

### CODE REVIEW

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -2661,7 +2661,11 @@ function oncontroller_testsetup()
     if [ -n "$want_docker" ] ; then
         image_name="cirros"
         ssh_user="cirros"
-        openstack image create --public --container-format docker --disk-format raw --property hypervisor_type=docker --copy-from http://$clouddata/images/docker/cirros.tar $image_name | tee glance.out
+        if openstack image list | grep -q $image_name ; then
```
Comment (vuntz) [inline scripts/qa_crowbarsetup.sh]: Probably need to grep for `"^${image_name}$"`

Comment (jsuchome) [inline scripts/qa_crowbarsetup.sh]: ah, true, 'cirros' might be there anyway

Comment (aplanas) [inline scripts/qa_crowbarsetup.sh]: Mostly `" ${image_name} "` or `"[[:space:]]${image_name[[:space:]]"`

**Comment by dirkmueller** [comment]
+1

**Comment by aplanas** [comment]
+1

**Pull request merged**